Simperium/simperium-ios
# Switching to WebKit

### Details:
In this PR we're dropping **UIWebView**, in favor of **WKWebView**. Plus;

- We're switching from **MobileCoreServices** to **CoreServices** (renamed SKD framework, triggering a warning)
- Updating Simpletodo sample projects, to use Launch.storyboards. Otherwise we're stuck with iPhone 5s screens!

@aerych it's been.. SO LONG since this lib was patched up!!!
Thanks in advance Eric!!

### Testing:
1. Open **samples/SimpletodoFinal.xcodeproj**
2. Launch the project
3. Press over the **Terms of Service** link

- [x] Verify a WebView shows up onscreen!
- [x] Verify the spinner on the top left corner stops after loading
